astropy/utils/decorators.py

```diff
@@ -452,14 +452,18 @@ def wrapper(*args, **kwargs):
                 if old_name[i] in kwargs:
                     value = kwargs.pop(old_name[i])
                     # Display the deprecation warning only when it's only
-                    # pending.
-                    if not pending[i]:
-                        warnings.warn(
-                            '"{0}" was deprecated in version {1} '
-                            'and will be removed in a future version. '
-                            'Use argument "{2}" instead.'
-                            ''.format(old_name[i], since[i], new_name[i]),
-                            AstropyDeprecationWarning, stacklevel=2)
+                    # not pending.
+                    if pending[i]:
+                        category = AstropyPendingDeprecationWarning
+                    else:
+                        category = AstropyDeprecationWarning
+
+                    warnings.warn(
+                        '"{0}" was deprecated in version {1} '
```

Comment:
Question: Given that `PendingDeprecationWarning` is silent (unless you explicitly catch it and turn it into exception), is it worth the extra logic to customize this message?